### PR TITLE
Update to PHPUnit v6 and remove root namespace slash in use statements

### PR DESCRIPTION
**Pull request by ziadoz**
This is just some prep work before I add some new bits and pieces.

### CODE REVIEW

**Comment by liamkeily** [comment]
Looks good to me 👍

File: tests/CategoriesTest.php [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <?php
 namespace DemandwareXml\Test;
 
-use \DemandwareXml\Assignment;
-use \DemandwareXml\Category;
-use \DemandwareXml\Document;
+use DemandwareXml\Assignment;
```
Comment (maxakropolis) [inline tests/CategoriesTest.php]: Should these be grouped?

Comment (ziadoz) [inline tests/CategoriesTest.php]: @maxakropolis I've fixed these.

**Comment by ziadoz** [comment]
@fusionspim/devs I've added a PHP CS Fixer configuration and applied the changes it found. I've also added that, plus a PHP syntax check to the build steps too.

**Comment by weshooper** [comment]
👍🏻

**Comment by maxakropolis** [comment]
👍

**Pull request merged**